vite.config.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'vite';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import react from '@vitejs/plugin-react';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: {
+      '@': Path2D.resolve(__dirname, './src'),
+    },
+  },
```

Comment:
**Suggestion:** Remove this line of code as it impedes me from running the app. 🤖

Reply:
I will work on that! 👨🏻‍💻🤓